### PR TITLE
made zapier integration more prominent

### PR DESCRIPTION
**Pull request by polluterofminds**
This PR makes the Zapier integration feel like a first-class feature. It moves the small mention of Zapier to a dedicated button that pops the user out to the specific Exceptionless Zapier integration page. 

This PR also updates the README to indicate that the dev should change into the `src` directory before installing dependencies and running grunt. 

Screen Shot 2021-03-31 at 8 25 27 AM

### CODE REVIEW

**Review by ejsmith — DISMISSED** [review]
LGTM

File: src/lang/en-us.json [hunk 1]
```diff
@@ -359,7 +360,8 @@
   "(Coming soon!)":"(Coming soon!)",
   "Add Slack Notifications":"Add Slack Notifications",
   "Remove Slack":"Remove Slack",
-  "Notice_Web_hooks":"The following web hooks will be called for this project. We also support <a href=\"http://zapier.com/\" target=\"_blank\">Zapier</a> which allows you to easily integrate Exceptionless with many different services (750+) like JIRA, HipChat, Twilio, Basecamp and more.",
+  "Notice_Web_hooks":"The following web hooks will be called for this project.",
+  "Notice_Zapier":"Exceptionless has a native Zapier integration. You can use Zapier to connect your Exceptionless account to over 3,000 other applications all without writing any code.",
```
Comment (niemyjski) [inline src/lang/en-us.json:364]: Can we also add these to the other language json. I just use google translate.

Comment (niemyjski) [reply src/lang/en-us.json:364]: Yep

File: src/lang/zh-cn.json [hunk 2]
```diff
@@ -360,6 +361,7 @@
     "Add Slack Notifications":"新增 Slack 通知",
     "Remove Slack":"移除 Slack",
     "Notice_Web_hooks":"此项目会调用下面表格中的Web Hook。与此同时我们还支持 <a href=\"http://zapier.com/\" target=\"_blank\">Zapier</a>，它可以更加容易地将 Exceptionless 与很多各种各样的服务（750+）进行整合，例如 JIRA、HipChat、Twilio、Basecamp 等等。",
```
Comment (niemyjski) [inline src/lang/zh-cn.json:363]: Need to delete the zapier part.

**Comment by niemyjski** [comment]
Merged Thanks for the PR!

**Pull request merged**